File: tests/integration/features/file_traceability/_language_parsers/c/02_c_function/file.c
#include <stdio.h>

/**
 * Some text.
 *
 * @relation(REQ-1, scope=function)
 */
void hello_world(void) {
    print("hello world\n");
}

/**
 * Some text.
 *
 * @relation(REQ-2, scope=function)
 */
void hello_world_2(void) {
    print("hello world\n");
}

/**
 * Some text.
 *
 * @relation(REQ-1, REQ-2, scope=function)
 */
void hello_world_3(void) {
    print("hello world\n");
}

/**
 * Some text.
 *
 * @relation(REQ-1, scope=function)
 * @relation(REQ-2, scope=function)
 */
void hello_world_4(void) {
    print("hello world\n");
}

File: tests/integration/features/file_traceability/_language_parsers/python/03_python_forward_function/file.py
def hello_world():
    """
    @relation(REQ-1, scope=function)
    """
    # @sdoc[REQ-1]
    print("hello world")  # noqa: T201
    # @sdoc(REQ-1)
    print("hello world")  # noqa: T201
    print("hello world")  # noqa: T201
    # @sdoc[/REQ-1]

def hello_world_2():
    print("hello world")  # noqa: T201
